### PR TITLE
VideoPress: Upload video from dashboard

### LINKED ISSUE #26316: VideoPress: uploading videos
Implement the uploading video workflow:

Image

Image

### PR DESCRIPTION
**Pull request by renatoagds**
~### ⚠️ Needs #26406~

Allowing user to upload videos from Dashboard.

Closes #26316 

#### Changes proposed in this Pull Request:

* Extract `useUploader` out of blocks.
* Add a new default hook for `useUploader`.
* Include new `thumbnail` labels in `VideoGrid`.

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion

N/A

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Start `VideoPress`.
* Navigate to dashbord.
* Click `Add new video` (if already has videos) | or use drop/add first video (if has no videos).
* It should upload new video.

### Demo

https://user-images.githubusercontent.com/1663717/192428049-acc3165d-bd32-4e5e-b9f6-6e74fa405afc.mp4

### CODE REVIEW

**Comment by nunyvega** [comment]
I just tested it and it works as expected.

The only thing that I noticed is that the `Select file to upload` under `Let's add your first video` does not work. It lets you select the file but it does not upload it. Dragging and dropping does work.


https://user-images.githubusercontent.com/16329583/192488083-517ce3da-ec04-415e-b3f9-44a2569fa7c3.mov

**Comment by nunyvega** [comment]
@renatoagds I just tested it after your last push and the first video upload works :)

File: projects/packages/videopress/src/client/admin/components/admin-page/index.tsx [hunk 1]
```diff
@@ -74,7 +139,15 @@ const Admin = () => {
 								<Text variant="headline-small" mb={ 3 }>
 									{ __( 'High quality, ad-free video', 'jetpack-videopress-pkg' ) }
 								</Text>
-								<Button fullWidth={ isSm }>{ addVideoLabel }</Button>
+								<FormFileUpload
+									onChange={ evt => handleFilesUpload( evt.currentTarget.files ) }
+									accept="video/*"
```
Comment (dhasilva) [inline projects/packages/videopress/src/client/admin/components/admin-page/index.tsx:144]: Should use the same extensions as the `VideoUploadArea` component, as not every video file is accepted (e.g. `.mkv`)

**Review by dhasilva — APPROVED** [review]
Golden! :tada:

**Review by retrofox — APPROVED** [review]
LGTM

**Comment by renatoagds** [comment]
Not diff because of: D88734-code

**Pull request merged**